zellij-utils/assets/config/log4rs.yml

```diff
@@ -0,0 +1,46 @@
+# Scan this file for changes every 30 seconds
+refresh_rate: 30 seconds
+
+appenders:
+  # An appender named "stderr" that writes to stderr
+  stderr:
+    kind: console
+    target: stderr
+
+  # default zellij appender, should be used across most of the codebase.
+  logFile:
+    kind: file
+    path: "zellij.log"
```

Comment:
We should look into setting this at runtime in Zellij. We already have a logging directory, so it would be nice if these logs were always written to wherever that happens to be.

Reply:
Sure, we can move logger initialization to runtime and get rid of this file entirely.